doc/content/guides/connecting-gateways/kerlinkwirnet/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Kerlink Wirnet Station"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-station/). This page guides you to connect it to The Things Stack.
+
+## Prerequisites
+
+1. User account on The Things Stack with rights to create Gateways and API Keys.
+2. Kerlink Wirnet Station:
+  1. For configuration via provisioning script: Common Packet Forwarder installed and enabled and/or gateway running firmware version `3.0` or higher.
+  2. For manual configuration: Common Packet Forwarder installed and enabled
```

Comment:
Putting this under prerequisites implies that this should already be setup and done before starting this guide, while the instructions follow

Reply:
Instructions follow for the `configuration via provisioning script`, I'm not sure what's the best approach here. We could drop the "manual" part and just let users use the script, but I feel that it'd be nice to have the "fields to set" regardless. Any suggestions?

Reply:
I think this just falls under a heading **Configuration** where people can choose what they want.

It's not a prerequisite.